src/REstate.Remote/Services/StateMachineService.cs

```diff
@@ -124,10 +136,10 @@ public class StateMachineService
                 request.MachineId,
                 input,
                 request.CommitTag,
-                Context.CallContext.CancellationToken);
+                GetCallCancellationToken());
         }
 
-        private static async Task<SendResponse> SendAsync<TState, TInput>(
+        public virtual async Task<SendResponse> SendAsync<TState, TInput>(
```

Comment:
Can you create an enhancement issue to address moving these methods to an internal interface for testability instead of public virtual. It won't be user visible, so it is fine, but want an item to track that tech debt.

Reply:
Issue created as #8